### ISSUE TITLE
Error importing tabular metadata

### ISSUE THREAD

**Issue by hackdna** [parklab/refinery-platform#1351]
Commit: e8b9869

### Steps to reproduce
Import a tabular metdata file (e.g., https://s3.amazonaws.com/data.cloud.refinery-platform.org/data/sample-data/human-chipseq/hg19-metadata-s3.txt: select SampleName column as Sample Identifier, DataFile as Data File Column and Organism as Species Column)

### Observed behavior
Error message displayed: "Oh no! We apologize. Something went terribly wrong. Please call your admin."
Browser console:
```
angular.js:10967 POST http://192.168.50.50/data_set_manager/import/metadata-table-form/ 500 (INTERNAL SERVER ERROR)
```
Refinery log:
```
[Fri Aug 19 14:34:38.094087 2016] [:error] [pid 25773:tid 140295893829376] Traceback (most recent call last):
[Fri Aug 19 14:34:38.094167 2016] [:error] [pid 25773:tid 140295893829376]   File "/home/vagrant/.virtualenvs/refinery-platform/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 112, in get_response
[Fri Aug 19 14:34:38.094263 2016] [:error] [pid 25773:tid 140295893829376]     response = wrapped_callback(request, *callback_args, **callback_kwargs)
[Fri Aug 19 14:34:38.094285 2016] [:error] [pid 25773:tid 140295893829376]   File "/home/vagrant/.virtualenvs/refinery-platform/local/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 22, in _wrapped_view
[Fri Aug 19 14:34:38.094294 2016] [:error] [pid 25773:tid 140295893829376]     return view_func(request, *args, **kwargs)
[Fri Aug 19 14:34:38.094301 2016] [:error] [pid 25773:tid 140295893829376]   File "/home/vagrant/.virtualenvs/refinery-platform/local/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
[Fri Aug 19 14:34:38.094308 2016] [:error] [pid 25773:tid 140295893829376]     return self.dispatch(request, *args, **kwargs)
[Fri Aug 19 14:34:38.094315 2016] [:error] [pid 25773:tid 140295893829376]   File "/home/vagrant/.virtualenvs/refinery-platform/local/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
[Fri Aug 19 14:34:38.094322 2016] [:error] [pid 25773:tid 140295893829376]     return handler(request, *args, **kwargs)
[Fri Aug 19 14:34:38.094329 2016] [:error] [pid 25773:tid 140295893829376]   File "/vagrant/refinery/data_set_manager/views.py", line 477, in post
[Fri Aug 19 14:34:38.094335 2016] [:error] [pid 25773:tid 140295893829376]     json.dumps({'error': error_msg}), 'application/json'
[Fri Aug 19 14:34:38.094342 2016] [:error] [pid 25773:tid 140295893829376]   File "/usr/lib/python2.7/json/__init__.py", line 243, in dumps
[Fri Aug 19 14:34:38.094349 2016] [:error] [pid 25773:tid 140295893829376]     return _default_encoder.encode(obj)
[Fri Aug 19 14:34:38.094356 2016] [:error] [pid 25773:tid 140295893829376]   File "/usr/lib/python2.7/json/encoder.py", line 207, in encode
[Fri Aug 19 14:34:38.094363 2016] [:error] [pid 25773:tid 140295893829376]     chunks = self.iterencode(o, _one_shot=True)
[Fri Aug 19 14:34:38.094369 2016] [:error] [pid 25773:tid 140295893829376]   File "/usr/lib/python2.7/json/encoder.py", line 270, in iterencode
[Fri Aug 19 14:34:38.094485 2016] [:error] [pid 25773:tid 140295893829376]     return _iterencode(o, 0)
[Fri Aug 19 14:34:38.094598 2016] [:error] [pid 25773:tid 140295893829376]   File "/usr/lib/python2.7/json/encoder.py", line 184, in default
[Fri Aug 19 14:34:38.094736 2016] [:error] [pid 25773:tid 140295893829376]     raise TypeError(repr(o) + " is not JSON serializable")
[Fri Aug 19 14:34:38.094752 2016] [:error] [pid 25773:tid 140295893829376] TypeError: IntegrityError('null value in column "task_id" violates not-null constraint\\nDETAIL:  Failing row contains (637, null, FAILURE, gAJVKkFuYWx5c2lzIEZhaWxlZCBkdXJpbmcgaW1wb3J0X2ZpbGUgc3VidGFza3EB..., 2016-08-19 18:34:38.085457+00, null, f, eJxrYKotZIzgYGBgSM7IzEkpSs0rZIotZC7WAwBWuwcA).\\n',) is not JSON serializable
```

### Expected behavior
File imported without errors. Also, if there is an error, the message should be less dramatic and more informative.

**Comment by jkmarx** [comment]
@hackdna Is there a reason this issue is in Barre and not Canton?

**Comment by hackdna** [comment]
Since it is a critical issue, I chose the most recent release that's still open.

**Comment by jkmarx** [comment]
@hackdna It was a mistake. We must have forgotten to close it on Tuesday.

**Comment by scottx611x** [comment]
Addressed by 29d22f5911b8578a246fcf39d4a6c8b251b648b9
